pkg/systemd/graphs.scss

```diff
@@ -19,6 +19,10 @@
     // Set the background of the container to the same as the toolbars…
     // mobile causes a wrap; we want toolbars to visually extend 100% wide
     background-color: var(--pf-c-page__main-section--m-light--BackgroundColor);
+    // Make the toolbar sticky, for mobile mode (when toolbars wrap)
+    position: sticky;
+    top: -3.5rem;
+    z-index: 100;
```

Comment:
Is this related to the scss transition, or did it just come up when testing pages? At first sight this could land right now, no?

Reply:
It's dependent on these changes but is an improvement as well.